integration_test/third_party_apps_test/applications/dcgm/metadata.yaml

```diff
@@ -115,8 +115,8 @@ expected_metrics:
       - *gpu_number_label
       - &direction_label
         name: direction
-        value_regex: transmit|receive
-        description: "Direction of the link traffic, one of [transmit, receive]."
+        value_regex: tx|rx
```

Comment:
Why didn't our existing tests fail here?

Reply:
Nevermind, I was using the "commits since last review" diff and I think there was an intermediate commit where this said `transmit|receive`.